x-pack/plugins/security_solution/common/api/detection_engine/prebuilt_rules/get_prebuilt_rules_and_timelines_status/get_prebuilt_rules_and_timelines_status_route.gen.ts

```diff
@@ -20,30 +20,30 @@ export const GetPrebuiltRulesAndTimelinesStatusResponse = z
     /**
      * The total number of custom rules
      */
-    rules_custom_installed: z.number().min(0),
+    rules_custom_installed: z.number().int().min(0),
```

Comment:
What change ended up causing these `int()`'s to be added here and in the `endpoint` files generated below? Not seeing a change to a `schema.yaml`, so looks like it was the `{{~#*inline "type_integer"~}}` change over in `zod_schema_item.handlebars`?

Reply:
While moving templates around, I noticed that we were missing `.int()` for integers, so I decided to add it right away. I also made some other minor clean-ups in the templates but they didn't affect generated artifacts.